src/Uno.UWP/Storage/StorageFile.Base.cs

```diff
@@ -15,8 +15,11 @@ partial class StorageFile
 	{
 		internal abstract class ImplementationBase
 		{
-			protected ImplementationBase(string path)
-				=> Path = path;
+			protected ImplementationBase()
```

Comment:
I would think that no matter the storage provider, a file can always have something equivalent to a path (the `Name` in you case), so I would really prefer if we were able to keep this strong concept.